src/testing/test-utils.test.ts

```diff
@@ -62,3 +62,78 @@ describe('selectOption', () => {
     expect(onChange).toHaveBeenCalledWith({ label: 'Bar', value: 'bar' }, expect.any(Object));
   });
 });
+
+describe('getSelectedOptions', () => {
+  it('returns empty list when no options are selected', () => {
+    // Arrange
+    const user = userEvent.setup();
+
+    render(
+      h(TextSelect, {
+        'aria-label': 'Test Select',
+        options: [
+          { label: 'Foo', value: 'foo' },
+          { label: 'Bar', value: 'bar' },
+          { label: 'Baz', value: 'baz' },
+        ],
+        value: null,
+      })
+    );
+
+    // Act
+    const select = new SelectHelper(screen.getByLabelText('Test Select'), user);
+    const selectedOptions = select.getSelectedOptions();
+
+    // Assert
+    expect(selectedOptions).toEqual([]);
+  });
+
+  it('returns empty list when no options are selected', () => {
```

Comment:
These three tests all have the same description ("returns empty list when no options are selected"), but they're not testing the same thing.